roadservice/core/models.py

```diff
@@ -720,3 +725,26 @@ def delete_mission_type(self, mission_type):
             mission_type.delete()
         except ProtectedError:
             raise BusyResourceError()
+
+
+class SMSSender(KavenegarAPI):
```

Comment:
The Google Code Style explains how to use camel case, and specially, how to deal with acronyms. The Python guide doesn't include this part, but the Java guide does:
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

By that convention, the class name should be `SmsSender`. But for example, `API` in `KavenegarAPI` or in Rest's `APIView` doesn't follow this convention. If it was Java, I would've insisted on changing the name. But now, do as you please :)